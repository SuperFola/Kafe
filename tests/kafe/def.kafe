x:int
x: int
x :int
x : int

x:int=1
x: int=1
x :int=1
x : int=1

x:int = 1
x: int = 1
x :int = 1
x : int = 1

cst x:int=1