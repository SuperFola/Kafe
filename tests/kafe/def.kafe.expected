(Program
    (Declaration (VarName x) (Type int))
    (Declaration (VarName x) (Type int))
    (Declaration (VarName x) (Type int))
    (Declaration (VarName x) (Type int))
    (Definition (VarName x) (Type int) (Integer 1))
    (Definition (VarName x) (Type int) (Integer 1))
    (Definition (VarName x) (Type int) (Integer 1))
    (Definition (VarName x) (Type int) (Integer 1))
    (Definition (VarName x) (Type int) (Integer 1))
    (Definition (VarName x) (Type int) (Integer 1))
    (Definition (VarName x) (Type int) (Integer 1))
    (Definition (VarName x) (Type int) (Integer 1))
    (ConstDef (VarName x) (Type int) (Integer 1))
)